UnicornRender/video/source/OrthographicCamera.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#include <unicorn/video/OrthographicCamera.hpp>
+
+#include <glm/gtc/matrix_transform.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+
+OrthographicCamera::OrthographicCamera(system::Window* window, glm::mat4& cameraProj) :
+    CameraProjection(window, cameraProj),
+    m_orthoScale(100.f),
+    m_scaledAspect(m_aspect / m_orthoScale),
+    m_right(static_cast<float>(window->GetSize().first) / 2),
+    m_top(static_cast<float>(m_pWindow->GetSize().second) / 2),
+    m_back(-1000.f),
+    m_front(1000.f)
```

Comment:
put all commas before each member so that when you add more variables, diff will be shorter